lib/web_ui/dev/chrome_installer.dart

```diff
@@ -241,9 +251,33 @@ Future<String> queryChromeDriverVersion() async {
   return chromeDriverVersion;
 }
 
+/// Make sure LUCI bot has the pinned Chrome version and return the executable.
+///
+/// We are using CIPD packages in LUCI. The pinned chrome version from the
+/// `browser_lock.yaml` file will already be installed in the LUCI bot.
+/// Verify if Chrome is installed and use it for the integration tests.
+String chromeExecutableForLUCI() {
```

Comment:
A better name would be `preinstalledChromeExecutable`. There's nothing LUCI-specific about this function, even if the _usage_ may be LUCI-specific (although I can imagine this being used in other contexts). The docs should also not talk about LUCI.

Reply:
done.